language-support/ts/codegen/tests/src/DA/Test/Daml2jsUtils.hs

```diff
@@ -52,3 +55,24 @@ setupYarnEnv rootDir (Workspaces workspaces) tsLibs = do
             , let pkgName = "@" <> T.replace "-" "/"  (T.pack name)
             ]
         ]
+
+testDependencies :: Value
+testDependencies = object
+    [ "devDependencies" .= object
+        [ ("@types/jest", String "^25.2.1")
+        , ("@types/node", String "^13.11.1")
+        , ("@types/puppeteer", String "^2.0.1")
+        , ("@types/wait-on", String "^4.0.0")
+        , ("puppeteer", String "^2.1.1")
+        , ("wait-on", String "^4.0.2")
+        ]
+    ]
+
+addTestDependencies :: FilePath -> IO ()
+addTestDependencies uiDir = do
+    packageJson <- BS.readFile (uiDir </> "package.json")
+    let packageJsonObj = case decode (BSL.fromStrict packageJson) of
+            Nothing -> error "Could not decode package.json"
+            Just pJ -> pJ
+        newPackageJson = lodashMerge packageJsonObj testDependencies
```

Comment:
Do we really need a loadash merge here? It seems simpler to just have a file that only lists the devDependencies and then do something like an `insertWith "devDependencies"`.

Reply:
That would work too. This seems fairly simple, though requires a new dependency on Aeson.Extra. Not sure if that's a problem. I am planning to get the extra dependencies from a file, though not sure where it should go. Maybe a supplemental directory in `templates` like `create-daml-app-test-resources`.